versioned_docs/version-v2.18.x/user-guide/install-zowe-server-install-wizard-1-0-0.md

```diff
@@ -0,0 +1,266 @@
+# Installing Zowe via Zowe Server Install Wizard
```

Comment:
@1000TurquoisePogs - Please adviser where this file should be located in the sidebar.

Reply:
it appears to be identical to the v3 equivalent page https://docs.zowe.org/stable/user-guide/install-zowe-server-install-wizard 
 
<img width="263" alt="image" src="https://github.com/user-attachments/assets/b6de1449-4f7f-46b9-b6b6-860c5f10c431" />